get.sh

```diff
@@ -555,7 +555,10 @@ getFunctionalTestMaterial()
 			if [[ "$dir" != "" ]] && [[ -d $dest/$dir ]]; then
 				echo "Stage $dest/$dir to $TESTDIR/$dir"
 				# already in TESTDIR, thus copy $dir to current directory
-				cp -r $dest/$dir ./
+					cp -r $dest/$dir ./
```

Comment:
Please undo this change in indentation at line 558. It's not required.

Reply:
Modified could you please check now.